optuna_dashboard/ts/components/StudyDetail.tsx

```diff
@@ -163,15 +180,40 @@ export const StudyDetail: FC<{
     )
   } else if (page === "trialList") {
     content = <TrialList studyDetail={studyDetail} />
-  } else if (page === "trialTable") {
+  } else if (page === "trialTable" && study !== null) {
+    const detailButtonGenerator = (studyId: number, trialNumber: number) => {
+      return (
+        <IconButton
+          component={Link}
+          to={URL_PREFIX + `/studies/${studyId}/trials?numbers=${trialNumber}`}
+          color="inherit"
+          title="Go to the trial's detail page"
+          size="small"
+        >
+          <LinkIcon />
+        </IconButton>
```

Comment:
How about moving this component to `tslib/react/src/components/TrialTable.tsx` and just passing the link by props?